src/tests/SharpRaven.UnitTests/SchemaHelper.cs

```diff
@@ -72,11 +72,18 @@ public static string GetSchemaPath()
             var directory = new DirectoryInfo(Environment.CurrentDirectory);
             FileInfo file = null;
 
+#if (net35)
+            while (directory != null && directory.Exists &&
+                   (file = directory.GetFiles("*.json")
+                       .FirstOrDefault(f => f.FullName.EndsWith("schema.json"))) == null)
+                            directory = directory.Parent;
+#else
```

Comment:
Same reason as the other comment, I thought you would prefer the original official .net implementation if it was available.